### ISSUE TITLE
Adjusted warming & cooling?

### ISSUE THREAD

**Issue by darchitectlp** [Stereowalker/Survive#183]
It's unclear how this works. Do all four armor slots have to have adjusted warming applied before you are immune to cold? Likewise, does adjusted cooling have to be applied to all armor pieces before being immune to heat? 

What about Warming I-VII? Do you have to have Warming VII on all four pieces before cold immunity? A few notes on your curse page or wiki here would help. Thanks for a fun mod!

**Comment by Stereowalker** [comment]
The adjusted warming only needs to be applied on one armor set to be invulnerable to cold and the regular warming enchantment should only be applied when and where you need it as it will still heat you up even if you are hot, the same applies to the cooling enchantments. Hope this answers your question

**Comment by darchitectlp** [comment]
Yes, thanks! Maybe add that to the wiki or something, so people understand how it works, and that the adjusted enchants only appear as treasure. Great mod!
